Controllers/Frontend/MakairaConnect.php

```diff
@@ -384,15 +407,15 @@ static function ($entity) {
         /** @var Mapper\EntityMapper $mapper */
         $mapper = $this->get('makaira_connect.mapper');
 
-        $changes = [
+        [
```

Comment:
This cannot be removed!
```suggestion
        $changes = [
```